tests/misc_env/install_prereq.py

```diff
@@ -80,10 +88,13 @@ def install_prereq(
         ceph.exec_command(cmd="sudo systemctl restart NetworkManager.service")
         if not skip_subscription:
             setup_subscription_manager(ceph)
-            if enable_eus:
-                enable_rhel_eus_rpms(ceph, distro_ver)
+            if not skip_enabling_rhel_rpms:
+                if enable_eus:
+                    enable_rhel_eus_rpms(ceph, distro_ver)
+                else:
+                    enable_rhel_rpms(ceph, distro_ver)
             else:
-                enable_rhel_rpms(ceph, distro_ver)
+                log.info("Skipped enabling the RPM's provided by the Subscription")
```

Comment:
this is a wrong message. subscription provides lot of rpms are not disabling all of them, we are skipping only rhel-rpms that's all, can you change the message here.

Reply:
Changed the logging with appropriate message